Gruntfile.js

```diff
@@ -21,8 +30,31 @@ module.exports = function(grunt) {
                    'icons/_base64-icons.scss' : 'icons/font/ddg-serp-icons.woff'
                 }
             }
-        }
+        },
+        webfont: {
+            icons: {
+                src: "icons/src/svgs/*.svg",
+                options: {
+                    dest: "icons/font/",
+                    destCss: "build",
+                    fontFilename: "ddg-serp-icons",
+                    types: "eot,woff,ttf,svg",
+                    codepoints: codepoints,
+                    customOutputs: [{
+                        template: "icons/src/templates/_icons.scss",
+                        dest: "mixins/_icons.scss"
+                    },{
+                        template: "icons/src/templates/_ddgsi.scss",
+                        dest: "icons/_ddgsi.scss",
+                        context: { version: Math.random() }
```

Comment:
So far we've had numeric versions (e.g. 125). We only really need that for cache busting. Using `Math.random()` basically makes sure the cache is busted every time the fonts are regenerated without us having to do anything.